packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -746,6 +746,21 @@ class SliverChildListDelegate extends SliverChildDelegate {
 }
 
 /// A base class for sliver that have [KeepAlive] children.
+/// It is an abstart class.
+///
+///
+/// See also:
+///
+///  * [KeepAlive], used with SliverWithKeepAliveWidgets as base parent,
+///     this widgets wraps around the child which needs to   
+///     be kept alive. Though it is not need when we use delegates.
+///  * [SliverChildBuilderDelegate] and [SliverChildListDelegate], which 
+///     are commonly used subclasses of [SliverChildDelegate] that use  
+///     a builder callback and an explicit child list, respectively.
```

Comment:
I don't think the reference to [SliverChildDelegate] here is needed, since it's rare that a Flutter dev need to know too much about it. But, let's keep the info about [SliverChildBuilderDelegate] and [SliverChildListDelegate] and keep it as closely related to the concept of "keep alive" as possible, since that's what our users would be most interested in.

```suggestion
///  * [SliverChildBuilderDelegate] and [SliverChildListDelegate], slivers which make use of
///     the keep alive functionality through the `addAutomaticKeepAlives` property.
```